chef/cookbooks/neutron/recipes/network_agents.rb

```diff
@@ -16,34 +16,13 @@
 
 include_recipe "neutron::common_agent"
 
-unless node[:neutron][:use_gitrepo]
-  pkgs = [ node[:neutron][:platform][:dhcp_agent_pkg], node[:neutron][:platform][:metering_agent_pkg] ]
-  pkgs.uniq.each { |p| package p }
+pkgs = [ node[:neutron][:platform][:dhcp_agent_pkg], node[:neutron][:platform][:metering_agent_pkg] ]
+pkgs.uniq.each { |p| package p }
```

Comment:
Wouldn't it be cleaner/nicer, and more in line with next package call, just to write
``package node[:neutron][:platform][:dhcp_agent_pkg]``
``package node[:neutron][:platform][:metering_agent_pkg]``

(missing ``uniq`` should not hurt)